Homework

webFarrock opened the pull request.

webFarrock committed:
basic project settings

webFarrock committed:
task01

webFarrock committed:
task02

webFarrock committed:
task03

webFarrock committed:
task04

webFarrock committed:
task05

webFarrock committed:
task06

webFarrock committed:
task08

webFarrock committed:
task08 upd tests with useFakeTimers

webFarrock committed:
task09

webFarrock committed:
task10

webFarrock committed:
task07

webFarrock committed:
install babel

webFarrock committed:
refactor for es6 modules

webFarrock committed:
tast07 fix test

webFarrock committed:
setup eslint

webFarrock committed:
eslint apply rules

webFarrock committed:
setup prettier

webFarrock committed:
install husky and lint-staged

webFarrock committed:
add sanity check

webFarrock committed:
try to break tests

webFarrock committed:
rollback broken test

AntiHero commented on src/task-07/script.js:
```diff
@@ -0,0 +1,75 @@
+
+  renderTextList(textBlockEl, textList);
+
+  appendBtnEl.addEventListener("click", () => updateTextBlock(appEl));
```
нужно добавить обработчик на инпут, который будет изменять свойство hidden кнопки в зависимости от наличия текста в input'e

webFarrock replied:
готово

webFarrock committed:
fixes after code review
https://github.com/webFarrock/otus-js-basics-lesson-06-hw/pull/1#discussion_r746811122

webFarrock committed:
lint error fix
